test/core/serialization/serialization_vectors_test.cpp

```diff
@@ -84,7 +84,7 @@ TEST(SerializationVectorsTest, SignedMessage) {
 }
 
 /// BlockHeader JSON and CBOR
-TEST(SerializationVectorsTest, BlockHeader) {
+TEST(SerializationVectorsTest, DISABLED_BlockHeader) {
```

Comment:
Please, leave comment why it is disabled.

Reply:
Serialization vectors repo didn't make updates for block yet, so this test is failing